docs/src/administration/web/configure-environment.md

```diff
@@ -50,84 +51,86 @@ There are also additional options:
-You can see the language version, the container size, the amount of persistent disk,
-the number of active workers and cron jobs, and the command to SSH into the container.
+For non-Production environments, you can [change their status](../../environments/deactivate-environment.md).
 
-You can also see all cron jobs with their name, frequency, and command.
+### Outgoing emails
 
-![Services: app overview](/images/management-console/service-tab/app-overview.png "0.5")
+Under **Outgoing emails**, you can allow your environment to [send emails](../../development/email.md):
 
-To see the YAML file that led to this configuration, click **Configuration**.
+![Environment email](/images/management-console/env-email.png "0.75")
 
-### Services
+### Hide from search engines
 
-To see more detailed information about a running service, click it in the tree or list.
+Under **Hide from search engines**, you can tell [search engines to ignore the site](../../environments/search-engine-visibility.md):
 
-The **Overview** gives you information about the given service.
-You can see the service version, the container size, and the disk size, if you've configured a persistent disk.
+![Environment search](/images/management-console/env-search.png "0.5")
 
-![Services: service overview](/images/management-console/service-tab/service-overview.png "0.5")
+### HTTP access control
 
-To see your `.platform/services.yaml` file that led to these routes, click **Configuration**.
+Under **HTTP access control**, you can [control access to your environment using HTTP methods](../../environments/http-access-control.md):
 
-## Environment settings
+![Settings control access with password and by IP](/images/management-console/settings-basics-access-control.png "0.5")
 
-To see settings for a given environment, click {{< icon settings >}} **Settings** within that environment.
+### Variables
 
-![Settings for an environment](/images/management-console/env-settings.png "0.75")
+Under **Variables**, you can define [environment variables](../../development/variables/_index.md):
 
-### Environment name
+![Configure Platform.sh environment variables](/images/management-console/settings-variables-environment.png "0.6")
 
-The first setting allows you to modify the name of the environment and view its parent environment.
+## Service information
 
-### Status
+For each environment, you can view information about how your routes, services, and apps are currently configured.
 
-The **Status** setting shows whether or not your environment is [active](../../other/glossary.md#active-environment).
-For non-Production environment, you can [change their status](../../environments/deactivate-environment.md).
+To do so, click **Services**.
+By default, you see configured routes.
 
-![Environment status](/images/management-console/env-status.png "0.5")
+### Routes
 
-### Outgoing emails
+The **Router** section shows a list of all the routes configured on your environment.
```

Comment:
:hammer_and_wrench: I think links to what each of these sections means might help, especially given that the target audience is people unfamiliar with the UI and Platform.sh in general.

```suggestion
The **Router** section shows a list of all the [routes configured on your environment](../../define-routes/_index.md).
```